.doc_gen/metadata/cross_metadata.yaml

```diff
@@ -289,3 +289,50 @@ cross_RekognitionVideoDetection:
             sqs:
   services:
     rekognition:
+cross_DetectFaces:
+  title: Detect faces in an image using an &AWS; SDK
+  title_abbrev: Detect faces in an image
+  synopsis_list:
+    - Save an image in an &S3long; (&S3;) bucket.
+    - Displays facial details, such as age range, gender, and emotion (smiling, etc.).
+  languages:
+    Rust:
+      versions:
+        - sdk_version: 1
+          block_content: cross_detect_faces_rust_block.xml
+  services:
+    s3: {}
+cross_DetectLabels:
+  title: Save EXIF and other image information using an &AWS; SDK
+  title_abbrev: Save EXIF and other image information
+  synopsis_list:
+    - Gets EXIF information from a a JPG, JPEG, or PNG file.
+    - Uploads the image file to an &S3long; (&S3;) bucket.
+    - Uses &REKlong; (&REK;) to identify the three top attributes (labels in &REK;) in the file.
+    - Adds the EXIF and label information to a &DDBlong; (&DDB;) table in the Region.
+  languages:
+    Rust:
+      versions:
+        - sdk_version: 1
+          block_content: cross_detect_labels_rust_block.xml
+  services:
+    dynamodb: {}
+    rekognition: {}
+    s3: {}
+cross_Telephone:
+  title: Synthesises a text file into an audio file and converts that back to text using an &AWS; SDK
+  title_abbrev: Synthesises a text file into an audio file and converts that back to text
```

Comment:
These titles are pretty long, and synthesize is an odd verb. Maybe shorten to something like:
`Convert text to speech and back to text...`